euphoria-hbase/src/main/java/cz/seznam/euphoria/hbase/HFileSink.java

```diff
@@ -99,6 +111,27 @@ public Builder withTable(String table) {
       return this;
     }
 
+    /**
+     * Set zookeeper quorum for the connection.
+     * @param quorum the quorum
+     * @return this
+     */
+    public Builder withZookeeperQuorum(String quorum) {
+      updaters.add(j -> j.getConfiguration().set(
```

Comment:
Is there any reason, this is not consistent with HBase(Source|Sink)? In terms of not using `updaters`